spec/miscellaneous/mailers/user_mailer_spec.rb

```diff
@@ -285,4 +285,37 @@
       end
     end
   end
+
+  describe "signup_notification" do
+    let(:user) { create(:user) }
+
+    subject(:email) { UserMailer.signup_notification(user.id).deliver }
+
+    # Test the headers
+    it_behaves_like "an email with a valid sender"
+
+    it "has the correct subject line" do
+      subject = "[#{ArchiveConfig.APP_SHORT_NAME}] Confirmation"
+      expect(email).to have_subject(subject)
+    end
+
+    # Test both body contents
+    it_behaves_like "a multipart email"
+
+    it_behaves_like "a translated email"
+
+    describe "HTML version" do
+      it "has the correct content" do
+        expect(email).to have_html_part_content("Once your account is up and running, you can post your fanworks, set up email")
+        expect(email).to have_html_part_content("follow this link to activate your account")
```

Comment:
The punctuation for "follow this link to activate your account" is a bit different in the two versions of the email. It might be worthwhile to specifically look for those differences in the tests, i.e. use `have_html_part_content("follow this link to activate your account.")` here and `have_text_part_content("follow this link to activate your account:")` in the text version.